app/partials/errorDetail.html

```diff
@@ -1,9 +1,12 @@
 <pagination>
 
 <div class="detail-header content-l">
-    <div class="content-l_col content-l_col-2-3">
+    <div class="content-l_col content-l_col-1-3">
         <pagination-size></pagination-size>
     </div>
+    <div class="content-l_col content-l_col-1-3">
+        <code-descriptions></code-descriptions>
+    </div>
```

Comment:
I'd shorten the text to just "HMDA Edit Values" with the full text as a tooltip. Also reduce the amount of space between it and the Export button even further; maybe just a 2em right margin for spacing?